### ISSUE TITLE
Cannot add events to the timeline

### ISSUE THREAD

**Issue by The-Toblin** [worldanvil/worldanvil-bug-tracker#334]
**World Anvil Username**:
Toblin

**Chronicle Title**:
Test History

**Chronos System**
Wut?

**Describe the Issue**
Creating new events inside the added timeline does not function. If I right click inside the area named "Main History", marked by yellow arrow, I get a browser context right click menu. If I click outside the black area into the gray area, marked by red arrow, I get "create new event". 

**To Reproduce**
See above

**Expected behavior**
Clicking inside the existing timeline should add an event to it. If this is not the method to create an event inside said timeline, explanation should be updated.
![ksnip_20210611-112528](https://user-images.githubusercontent.com/79937532/121664672-014be680-caa8-11eb-80bf-8275b0f3150c.png)

**Comment by The-Toblin** [comment]
Clarification: The black banner may just be a "banner", a name tag. That's not visually clear. It looks like it's an actual timeline you should click.

**Comment by Xuroth** [comment]
Yes, the black bar is just a banner. I will update it to have timeline managment options in the future (and expand explanation)
I'll check into the context menu issues.

**Comment by Kordishal** [comment]
Also in general creating a new event is non obvious. There should be a hint somewhere that tells you to create new events & locations with right click. Like the warning.

**Comment by KajetanWrites** [comment]
> Also in general creating a new event is non obvious. There should be a hint somewhere that tells you to create new events & locations with right click. Like the warning.

Yes, I agree. It took me a while to find it
